numpy/testing/nosetester.py

```diff
@@ -326,9 +325,9 @@ def test(self, label='fast', verbose=1, extra_argv=None,
              <http://nedbatchelder.com/code/modules/coverage.html>`_).
         raise_warnings : str or sequence of warnings, optional
             This specifies which warnings to configure as 'raise' instead
-            of 'warn' during the test execution.  Valid strings are:
+            of 'always' during the test execution.  Valid strings are:
 
-              - "develop" : equals ``(DeprecationWarning, RuntimeWarning)``
+              - "develop" : equals ``(Warning,)``
```

Comment:
Thought about it some more, and wondering about a middle way, also because it should avoid possible problems with scipy. just leave develop as is, and introduce a new flag `"all"`. Then we could still run some of the Travis tests in "all" mode to ensure that we don't miss odd new warnings/keep the warnings tidy.